### PR TITLE
add release notes for 202301

### PR DESCRIPTION
**Pull request by powdercloud**
CrUX Release notes for 202301.

### CODE REVIEW

**Comment by powdercloud** [comment]
@rviscomi :-)

File: site/en/docs/crux/release-notes/index.md
Comment (rviscomi) [inline site/en/docs/crux/release-notes/index.md:55]:
```suggestion
 : - This month we've launched a [CrUX History API](/docs/crux/history-api/), which provides 25 weeks of historical data at both origin and URL level. We've [written a post](/blog/chrome-ux-report-history-api/) detailing how to use this new API, and a [Colab](https://colab.sandbox.google.com/github/GoogleChrome/CrUX/blob/main/colab/crux-history-api.ipynb) showing you an example of how to plot this historical data in various graphs.
```

**Review by rviscomi — APPROVED** [review]

**Review by AaronForinton — APPROVED** [review]

**Comment by powdercloud** [comment]
@AaronForinton @rviscomi Thanks! I don't have write access so I can't merge it myself. So please feel free to do that when you get a chance.

**Pull request merged**
